templates/application-kube-prometheus-stack.yaml

```diff
@@ -91,6 +92,22 @@ spec:
                     requests:
                       storage: {{ .Values.prometheus.volume_claim_storage_request}}Gi
             hostNetwork: {{ .Values.host_network.enabled }}
+            replicaExternalLabelName: __replica__
+            prometheusExternalLabelName: cluster
+            remoteWriteDashboards: true
+            additionalRemoteWrite: 
+            - url: https://cortex-metrics.apps.prod.glueops.onglueops.rocks/api/v1/push
```

Comment:
If we end up changing the path on the internal deployment this will need to be updated as well